src/lib/components/Tag.svelte

```diff
@@ -1,21 +1,56 @@
 <script lang="ts">
-  export let tagName: "span" | "h3" | "li" = "span";
+  export let tagName: "span" | "li" = "span";
+  export let intent: "warning" | "success" | "error" | "info" = "info";
```

Comment:
nitpick: not a strong opinion but how about not having a default value for the intent? Otherwise developer should be aware of it.

Reply:
Good point, I wanted to keep backwards compatibility though. So that we don't have to change the code.